project/src/engines/hoomd/project.py

```diff
@@ -32,34 +32,60 @@ def run_hoomd(job):
     structure = ff.apply(filled_box)
 
     write_gsd(structure, job.fn("init.gsd"), ref_distance=rd, ref_energy=re)
-    # NOTE: create_hoomd3_forcefield depends on PR
-    # https://github.com/mosdef-hub/mbuild/pull/871
+    # ref_distance: angstrom -> nm
+    # ref_energy: kcal/mol -> kJ/mol
+    # ref_mass: dalton -> amu
     snapshot, forcefield, ref_vals = create_hoomd3_forcefield(
-        structure, ref_distance=10, ref_energy=1 / 4.184
+        structure, ref_distance=0.1, ref_energy=1 / 4.184, ref_mass=0.9999938574
     )
 
     device = hoomd.device.auto_select()
-    simulation = hoomd.Simulation(device=device, seed=job.sp.replica)
-    simulation.create_state_from_snapshot(snapshot)
+    sim = hoomd.Simulation(device=device, seed=job.sp.replica)
+    sim.create_state_from_snapshot(snapshot)
     gsd_writer = hoomd.write.GSD(
-        filename=job.fn("traj.gsd"),
+        filename=job.fn("trajectory.gsd"),
         trigger=hoomd.trigger.Periodic(10000),
         mode="ab",
     )
-    simulation.operations.writers.append(gsd_writer)
+    sim.operations.writers.append(gsd_writer)
+
+    logger = hoomd.logging.Logger(categories=["scalar"])
+    logger.add(sim, quantities=["timestep", "tps"])
+    thermo_props = hoomd.md.compute.ThermodynamicQuantities(
+        filter=hoomd.filter.All()
+    )
+    sim.operations.computes.append(thermo_props)
+    logger.add(
+        thermo_props,
+        quantities=[
+            "kinetic_energy",
+            "potential_energy",
+            "pressure",
+            "kinetic_temperature",
+            "volume",
+        ],
+    )
+    file = open("log.txt", mode="x", newline="\n")
+    table_file = hoomd.write.Table(
+        output=file,
+        trigger=hoomd.trigger.Periodic(period=5000),
+        logger=logger,
+        max_header_len=7,
+    )
+    sim.operations.writers.append(table_file)
+
     integrator = hoomd.md.Integrator(dt=0.005)
     integrator.forces = forcefield
     nvt = hoomd.md.metehods.NVT(
         filter=hoomd.filter.All(), kT=job.sp.temperature, tau=1.0
```

Comment:
We may have to convert the temperatures here into energy units.

Reply:
good point!

Reply:
Unyt should be able to handle that for us. https://unyt.readthedocs.io/en/stable/modules/unyt.equivalencies.html#unyt.equivalencies.ThermalEquivalence


```python3
import unyt as u
temp = 300 * u.K
energy_temp = temp.to_equivalent('eV', 'thermal')
```

In this case, I get for 300K, a thermal energy equivalent to ~0.025 eV which is what [wikipedia reports in the "Temperature" section](https://en.wikipedia.org/wiki/Electronvolt)

Reply:
Not entirely sure what else is needed here to make it unitless like hoomd expects? @tcmoore3 or is converting it to energy units sufficient?

Reply:
That's awesome! In this case, we'd want to convert to kJ/mol, right?

Reply:
@justinGilmer I think if all the units (distance/mass/energy) are converted by the `create_hoomd_forcefield`, keeping the kT energy value consistent should do it.

Reply:
Yes, I think that unyt code is what we want. In this case, it would be something like:
```python
>>> import unyt as u
>>> temp = 300 * u.K
>>> kT = temp.to_equivalent('kJ/mol', 'thermal')
>>> kT
unyt_quantity(2.49433864, 'kJ/mol')
```

Reply:
Perfect! Thanks for testing that too!